### ISSUE TITLE
Feature request: split editor

### ISSUE THREAD

**Issue by richardxu100** [careteditor/caret#707]
Previously, I was able to create a new window with cmd-shift-n, but it seems like this feature has been removed in the latest release. I like to keep two windows open—one with notes and the other blank when writing essays. Are there plans to bring this feature back? If not, is there a way to downgrade my app to an earlier version?

**Comment by erusev** [comment]
Hi @richardxu100, the multiple windows support is a tough feature that takes a significant part of our energy and since already have features like tabs and Recent Folders (^R on the mac) we were wondering if it is still justifiable for us to spend that much energy on it.

Could you share your use cases for multiple windows and why it's important for you?

Thanks!

**Comment by richardxu100** [comment]
Thanks @erusev for responding to me! I only purchased caret a few days ago, but have already fell in love with how efficient and fun it is to write in it. I previously used Byword to write essays, which allowed me to keep multiple windows open—one for notes, and the other for writing. And I did the same with Caret a few days ago:  

<img width="1280" alt="screen shot 2018-03-29 at 9 27 10 am" src="https://user-images.githubusercontent.com/13954830/38091578-e85ee208-3333-11e8-90b9-cffd42b90632.png">

What I love about Caret is the ability to quickly scroll to different heading using cmd-R—it makes referencing notes super efficient. I find a lot of value from multiple windows, but would understand if that feature were taken away because of its cost. (I currently have Caret 3.0 installed to use as a second window!)

**Comment by erusev** [comment]
Thanks, @richardxu100 for the explanation!

I have a couple of questions:

1. Have you tried to use separate tabs  <kbd>⌘</kbd> <kbd>⇧</kbd> <kbd>[</kbd> or <kbd>]</kbd> or cm <kbd>⌘</kbd> <kbd>Number</kbd> to quickly navigate between tham?
2. Wouldn't something like split editor, as in Sublime text, be a better solution than multiple windows?

Thanks!

**Comment by richardxu100** [comment]
Hey @erusev, thanks for these suggestions! I didn't know about cmd-number as a shortcut, so this definitely helps! Is this what split editor looks like in sublime? 

<img width="1277" alt="screen shot 2018-03-30 at 3 34 56 pm" src="https://user-images.githubusercontent.com/13954830/38151184-3c9ff468-3430-11e8-925d-14bc97e74c3e.png">

If so, I think this feature would be a great substitute for multiple windows. Thank you again for your help!

**Comment by erusev** [comment]
Yes, that's how it would work. Thanks for getting in touch and for sharing your thoughts! Please, let us know if you come up with more ideas on how to improve the app!

**Comment by bellebethcooper** [comment]
Just wanted to add that I'd find this split view really handy, too. One of my regular uses of Caret is a monthly report where I need to reference last month's report as I write the current one. I use Better Touch Tool to add swipes for quickly switching tabs, but having them both visible would make this process a lot smoother.
